code/modules/mob/living/silicon/mommi/mommi_modules.dm

```diff
@@ -96,3 +92,21 @@
 		"Gravekeeper" = "cogspider"
 		)
 	law_type = "Gravekeeper"
+
+/obj/item/weapon/robot_module/mommi/sammi
+	name = "stationary assembly mmi robot module"
+
+/obj/item/weapon/robot_module/mommi/sammi/New(var/mob/living/silicon/robot/R)
+	..()
+	for(var/module in modules)
```

Comment:
I still believe this is a crime against OOP and a clear signal that this module shouldn't be a child of mommi

Reply:
Made this a bit less bad with the cost of redundant vars code.